perl/imap/Cyrus/IndexFile.pm

```diff
@@ -1381,11 +1381,12 @@ sub _make_fields {
 sub _make_pack {
   my $format = shift;
   my $size = shift;
+  my $name = shift;
   if ($format eq 'int32' or $format eq 'time_t') {
     return 'N';
   }
-  elsif ($format eq 'int64' or $format eq 'time64') { # ignore start..
-    return 'x[N]N';
+  elsif ($format eq 'int64' or $format eq 'time64') {
+    return 'Q>';
```

Comment:
I can't believe you made me have to go read the pack docs.

I believe the meaning changed from "skip one long, read the next" to "read a whole quad".  Makes sense. LMK if I got it wrong.